storybook/pages/TokenListPopupPage.qml

```diff
@@ -0,0 +1,147 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Storybook 1.0
+import Models 1.0
+import SortFilterProxyModel 0.2
+
+import StatusQ.Core.Utils 0.1
+
+import AppLayouts.Profile.popups 1.0
+
+SplitView {
+    id: root
+    Logs { id: logs }
+
+    readonly property var sourcesOfTokensModel: SourceOfTokensModel {}
+    readonly property var flatTokensModel: FlatTokensModel {}
+    readonly property var tokensProxyModel: SortFilterProxyModel {
+        sourceModel: flatTokensModel
+
+        // TODO: This proxyRoles definition can be done using new component `LeftJoinModel` and from there build `chainName` and `explorerUrl`
+        proxyRoles:  [
+            ExpressionRole {
+                function getChainName(chainId) {
+                    return NetworksModel.getChainName(chainId)
+                }
+
+                name: "chainName"
+                expression: getChainName(model.chainId)
+
+            },
+            ExpressionRole {
+                name: "explorerUrl"
+                expression: { return  "https://status.im/" }
+            },
+            ExpressionRole {
+                name: "jsArraySources"
+                expression: model.sources.split(";")
+            }
+        ]
+    }
+
+    property var currentSourceItem
+
+    SplitView {
+        orientation: Qt.Vertical
+        SplitView.fillWidth: true
+
+        Item {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
+
+            PopupBackground {
+                anchors.fill: parent
+            }
+
+            Button {
+                anchors.centerIn: parent
+                text: "Reopen"
+
+                onClicked: dialog.open()
+            }
+
+            TokenListPopup {
+                id: dialog
+
+                anchors.centerIn: parent
+                closePolicy: Popup.NoAutoClose
+                visible: true
+                modal: false
+
+                sourceName: currentSourceItem ? currentSourceItem.name : ""
```

Comment:
Similar approach as in `SupportedTokenListPanel` could be used here. Then it will be simpler and `SourceOfTokensModel {}` doesn't need to have additional `load` signal.